articles/test/test_views.py

```diff
@@ -0,0 +1,250 @@
+import json
+from typing import Any
+from unittest.mock import patch
+
+from django.contrib.auth import get_user_model
+from django.test.client import BOUNDARY, MULTIPART_CONTENT, encode_multipart
+from faker import Faker
+from rest_framework import status
+from rest_framework.reverse import reverse
+from rest_framework.test import APIClient, APITestCase
+
+from articles.models import Article
+from articles.test.mocks import sample_image
+
+fake = Faker()
+User = get_user_model()
+
+
+class TestArticleViews(APITestCase):
+    def setUp(self) -> None:
+        self.client = APIClient()
+        self.password = fake.password()
+        self.user = User.objects.create_user(
```

Comment:
the problem with creating a user in setUp method is that an instance will be created before each test in this test case, 
 use setUpClass instead

Reply:
ok